USAGE.md

```diff
@@ -1017,8 +1017,7 @@ or with pipelining...
 
 ```powershell
 $asset | Remove-GitHubReleaseAsset -force
-
-----------
+```
```

Comment:
We actually want both.  That's the last example for Releases before we move on to Gists, and we use the `--------` to separate sections.

I'm having trouble creating the suggested fix for this part due to the fact that the fix includes the "```" characters, but what we want is the three "````", then a line feed, then the dashes, then another line feed, and then the `### Gists`